### PR TITLE
Exams, Grade Breakdown

### PR DESCRIPTION
**Pull request by cxuc163**

### CODE REVIEW

File: contents/handbook-exams.html [hunk 1]
```diff
@@ -1 +1,34 @@
-'content goes here' 
+<ul>
+  <li>No midterm.</li>
+  <li>Final exam
+    <ul>
+      <li>2 hours, open book, essay type questions (no MCQ), covers materials from all lectures, handouts, and tutorials.</li>
+      <li>The format is different from traditional exams. Samples: AY14/15 Sem 2 (
+      <a href="https://www.google.com/url?q=https%3A%2F%2Fwww.comp.nus.edu.sg%2F~cs2103%2FAY1516S1%2Ffiles%2F1415SEM2-CS2103-T.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNFAuYSQx2Q6o9vMlhAHa_5X-n7lhQ">exam paper</a>,
```
Comment (damithc) [inline contents/handbook-exams.html]: Clean up these links to remove extra stuff added by Google docs. For example, this link should be 
`http://www.comp.nus.edu.sg/~cs2103/AY1516S1/files/1415SEM2-CS2103-T.pdf`

File: contents/handbook-exams.html [hunk 2]
```diff
@@ -1 +1,34 @@
...
+      <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.comp.nus.edu.sg%2F~cs2103%2FAY1516S1%2Ffiles%2F1314SEM2-CS2103-T%2520comments.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNGxeWytJOR2efFF7WMUuyzGWckduA">comments about answers</a>) 
+      <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.comp.nus.edu.sg%2F~cs2103%2FAY1516S1%2Ffiles%2F1213SEM1-CS2103-T.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHTFlBWOAbCpND18uqsJb06wT9Zew">AY12/13-Sem1</a> 
+      <a href="http://www.google.com/url?q=http%3A%2F%2Fwww.comp.nus.edu.sg%2F~cs2103%2FAY1516S1%2Ffiles%2F1112SEM1-CS2103-T.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHQsfUY1GrAH2uuWHfHLZw_YA9ztg">AY11/12-Sem1</a>.</li>
+      <li>Attend the last lecture to learn more about the exam paper.</li>
+      <li>There will be no model answers for past papers. However, we can give feedback for answers you come up with.</li>
+      <li>Exam consultations (consultations near the exam) are discouraged but allowed. We prefer you to learn and clarify along the way, not at the end.</li>
```
Comment (damithc) [inline contents/handbook-exams.html]: Try not to exceed line length of 110-120 characters.

**Comment by damithc** [comment]
Some minor changes suggested.

**Comment by cxuc163** [comment]
modify those things, add participation points, appendixA

File: contents/handbook-appendixA.html [hunk 3]
```diff
@@ -1 +1,73 @@
...
+This means,</p>
+<ul>
+  <li>We may not take the most efficient route to building the software product. We take the 
+  route that allows you to learn the most.</li>
+  <li>Building a software product that is unique, creative, and shiny is not our priority 
+  (although we try to do a bit of that too). Learning to take pride in, and discovering the joy 
+  of, high quality software engineering work is our priority.</li>
+</ul>
+<p>Following from that, we evaluate you on not just much you&rsquo;ve done, but also, how well 
+you&rsquo;ve done those things.</p>
```
Comment (damithc) [inline contents/handbook-appendixA.html]: do we need `&rsquo;` here?

Comment (damithc) [inline contents/handbook-appendixA.html]: May be replace all `&rsquo;` and `&lsquo;` with regular quotes?

File: contents/handbook-policy-reuse.html [hunk 4]
```diff
@@ -1 +1,31 @@
...
+  <li>You are allowed to reuse work from external sources, subject to following conditions:
+    <ul>
+      <li>The work comes from a source of 'good standing' (such as an established open source 
+	  project). This means you cannot reuse code written by an outside 'friend'.</li>
+      <li>You clearly give credit to the original author. Acknowledge use of third party resources 
+	  clearly e.g. in the welcome message, splash screen (if any) or under the &lsquo;about&rsquo; 
+	  menu. If you are open about reuse, you are less likely to get into trouble if you 
+	  unintentionally reused something copyrighted.</li>
+      <li>You do not violate the license under which the work has been released. Please <span class
+	  ="highlighted">do not use 3rd-party images/audio</span> in your software unless they have
```
Comment (damithc) [inline contents/handbook-policy-reuse.html]: Insert line breaks in a way readability is maintained. Breaking a line in the middle of an attribute definition reduces readability.

**Comment by cxuc163** [comment]
The above problems have been fixed. All parts are finished.

File: contents/handbook-appendixD.html [hunk 5]
```diff
@@ -1 +1,69 @@
...
+    <p> Tip: make sure you know what these stand for: RTFM, STFW, GIYF</p>
+  </blockquote>
+<ul>
+  <li>Request our help: Failing all above, request for help using weekly 'doubts and help requests' sessions hosted on TEAMMATES (session open Wed-Fri weekly)</li>
+</ul>
+<p>Resources</p>
+<ul>
+  <li>[lifehacker article] <a href="http://lifehacker.com/google-tips-and-tricks-every-student-should-know-1508121671">Google Tips and Tricks Every Student Should Know</a></li>
+  <li>[Article] <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a> by Eric Steven Raymond</li>
+</ul>
```
Comment (damithc) [inline contents/handbook-appendixD.html]: Lines too long in this file?

File: contents/handbook-appendixE.html [hunk 6]
```diff
@@ -1 +1,96 @@
...
+<ul>
+  <li>2.1: Ravi implements a new feature in his local repo.</li>
+  <li>2.2: Ravi pulls the latest code from the main repo, merges it with his code, tests his new code to ensure it works with the latest version of the product, and pushes his 
+  code to his own fork. He then creates a<a href="https://help.github.com/articles/using-pull-requests/">pull request</a> to inform Adam that the new code is ready.</li>
+  <li>2.3: Adam pulls from Ravi's fork, merges with existing code, and run the unit tests.<br />
+    (a) If the tests fail, or if the new code is not up to the standard, he reverts back to the previous version, effectively rejecting Ravi's code. He adds a comment in the 
+	pull request to say the code has been rejected.<br />
+    (b) If Ravi's code is working and of acceptable quality, he pushes it to the main repo and closes the pull request.  </li>
+</ul>
+<p><img src="../images/github.png" width="533" height="503" /></p>
```
Comment (damithc) [inline contents/handbook-appendixE.html]: And this one?

**Comment by damithc** [comment]
Lines need shortening in two files. Otherwise looks fine.

**Comment by cxuc163** [comment]
Have fixed them.

**Pull request merged**